client/src/views/landing.vue

```diff
@@ -13,8 +13,9 @@
               </template>
               <p>The Name Request website is available as a Beta version. Name Requests obtained through the Beta are
                 official name requests and can be used in the province of British Columbia.</p>
-              <p>As part of a new <u>agile software development process</u>, the Name Request website will be
-                continually updated and improved based on feedback from citizens and businesses in BC.</p>
+              <p>As part of a new <a class="" :href="agileUrl">agile software development process</a>, the Name Request
```

Comment:
What's the URL?
Should "agile" be "Agile"?

Reply:
Oops will do!
Design shows it as lower case. Should it be capitalized?

Reply:
Fixed ( using vuetify color class helpers... ) I keep forgetting about these. Like i said, changed my life lol